<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCommands</class>
 <widget class="QDialog" name="DialogCommands">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Commands</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Message</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="pushButtonShowText">
              <property name="text">
               <string>Show text...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonInputNumber">
              <property name="text">
               <string>Input number...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonEndGame">
              <property name="text">
               <string>Game Over</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonChangeVariables">
              <property name="text">
               <string>Change variables...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonWhile">
              <property name="text">
               <string>Loop</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonWhileBreak">
              <property name="text">
               <string>Break loop</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonCondition">
              <property name="text">
               <string>Condition...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonOpenMainMenu">
              <property name="text">
               <string>Open main menu</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonOpenSavesMenu">
              <property name="text">
               <string>Open saves menu</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonModifyInventory">
              <property name="text">
               <string>Modify inventory...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonModifyTeam">
              <property name="text">
               <string>Modify team...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonStartBattle">
              <property name="text">
               <string>Start a battle...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonChangeState">
              <property name="text">
               <string>Change state...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSendEvent">
              <property name="text">
               <string>Send event...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonTeleportObject">
              <property name="text">
               <string>Teleport object...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonMoveObject">
              <property name="text">
               <string>Move object...</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Others</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
